fix http Microsoft links (#313)
MicrosoftDocs/iis-docs: iis/media/windows-media-services/windows-media-server-or-web-server.md

====================
by Dave Nelson

> You can deliver digital media audio and video files and live streams either from a [Windows Media server](https://technet2.microsoft.com/windowsserver2008/en/servermanager/streamingmediaservices.mspx) (a server that is running Windows Media Services) or from a Web server (a server that is running Internet Information Services (IIS)). Clients use different methods to receive digital media content from Windows Media servers and Web servers: they stream digital media from a Windows Media server and download digital media from a Web server.

Reviewer: ```suggestion
You can deliver digital media audio and video files and live streams either from a Windows Media server (a server that is running Windows Media Services) or from a Web server (a server that is running Internet Information Services (IIS)). Clients use different methods to receive digital media content from Windows Media servers and Web servers: they stream digital media from a Windows Media server and download digital media from a Web server.
```

> You can deliver digital media audio and video files and live streams either from a [Windows Media server](https://technet2.microsoft.com/windowsserver2008/en/servermanager/streamingmediaservices.mspx) (a server that is running Windows Media Services) or from a [Web server](https://technet2.microsoft.com/windowsserver2008/en/servermanager/webserver.mspx) (a server that is running Internet Information Services (IIS)). Clients use different methods to receive digital media content from Windows Media servers and Web servers: they stream digital media from a Windows Media server and download digital media from a Web server.

Reviewer 2: Remove link

Reviewer: Using suggestions to make it easier to apply the changes

```suggestion
You can deliver digital media audio and video files and live streams either from a [Windows Media server](https://technet2.microsoft.com/windowsserver2008/en/servermanager/streamingmediaservices.mspx) (a server that is running Windows Media Services) or from a Web server (a server that is running Internet Information Services (IIS)). Clients use different methods to receive digital media content from Windows Media servers and Web servers: they stream digital media from a Windows Media server and download digital media from a Web server.
```

User Datagram Protocol (UDP) has long been the preferred transport protocol for traditional streaming media. It is a fast, lightweight protocol without any retransmission or data-rate management functionality, making it an ideal protocol for transmitting real-time audio and video data, which can tolerate some lost packets. UDP also has higher priority than Transmission Control Protocol (TCP)-based HTTP for Internet traffic, giving streaming audio and video data higher priority over file and Web page transfers and increasing the likelihood of uninterrupted viewing on congested networks. A Windows Media server also uses UDP Resend, an intelligent UDP-retransmission scheme that ensures that it only retransmits lost packets that can be sent to a player in time to be played, instead of the blind retransmission scheme employed by TCP. This smart-resend feature conserves additional bandwidth on congested networks.

> A Windows Media server may stream digital media by using the Real Time Streaming Protocol (RTSP) with UDP-based transport (RTSPT) to take advantage of UDP-based streaming efficiencies. On networks that don't support UDP, the server is normally configured to use a process called [protocol rollover](https://technet2.microsoft.com/windowsserver2008/en/library/82fb84a9-96f5-49dc-92d2-cf76c912aab81033.mspx?mfr=true) to try TCP-based streaming, first by using RTSP, and if that doesn't work, it uses its own version of HTTP for firewalls that allow Web traffic through port 80. This enables corporate users to view Internet content without compromising firewall security and ensures that all users on all networks can access all streaming media content. For more information about the streaming protocols that are used by Windows Media Services, see [Comparing HTTP Streaming Protocol with RTSP](https://msdn.microsoft.com/library/bb905764.aspx).

Reviewer: ```suggestion
A Windows Media server may stream digital media by using the Real Time Streaming Protocol (RTSP) with UDP-based transport (RTSPT) to take advantage of UDP-based streaming efficiencies. On networks that don't support UDP, the server is normally configured to use a process called [protocol rollover](https://docs.microsoft.com/windows/desktop/wmformat/protocol-rollover) to try TCP-based streaming, first by using RTSP, and if that doesn't work, it uses its own version of HTTP for firewalls that allow Web traffic through port 80. This enables corporate users to view Internet content without compromising firewall security and ensures that all users on all networks can access all streaming media content. For more information about the streaming protocols that are used by Windows Media Services, see [Comparing HTTP Streaming Protocol with RTSP](https://docs.microsoft.com/previous-versions/windows/embedded/ee485306(v%3dwinembedded.60)).
```

## IIS Media Services

> IIS Media Services extensions are designed to add media delivery capabilities to a Web server that is running [Internet Information Services (IIS) 7](https://technet2.microsoft.com/windowsserver2008/en/servermanager/webserver.mspx). The IIS Media Services extensions take advantage of the broader market availability of high-bandwidth networks, which make a Web server an increasingly practical option for digital media content delivery. This section describes the following streaming and bandwidth management extensions that are available in IIS Media Services:

Reviewer: ```suggestion
IIS Media Services extensions are designed to add media delivery capabilities to a Web server that is running Internet Information Services (IIS) 7. The IIS Media Services extensions take advantage of the broader market availability of high-bandwidth networks, which make a Web server an increasingly practical option for digital media content delivery. This section describes the following streaming and bandwidth management extensions that are available in IIS Media Services:
```

| Delivers content through most [firewalls](https://www.microsoft.com/technet/security/guidance/networksecurity/firewall.mspx) | ![](windows-media-server-or-web-server/_static/image7.gif) | ![](windows-media-server-or-web-server/_static/image8.gif) |
| Streams without downloading | ![](windows-media-server-or-web-server/_static/image9.gif) | ![](windows-media-server-or-web-server/_static/image10.gif) |
| Broadcasts live content | ![](windows-media-server-or-web-server/_static/image11.gif) | **2![](windows-media-server-or-web-server/_static/image12.gif)** |

> | Delivers content using [multicast streaming](https://technet2.microsoft.com/windowsserver2008/en/library/0e5fa6c2-72ad-4337-8c85-e0fe5e85f8e41033.mspx?mfr=true) | ![](windows-media-server-or-web-server/_static/image13.gif) | **3![](windows-media-server-or-web-server/_static/image14.gif)** |

Reviewer: ```suggestion
| Delivers content using multicast streaming | ![](windows-media-server-or-web-server/_static/image13.gif) | **3![](windows-media-server-or-web-server/_static/image14.gif)** |
```

| Delivers content to iOS devices (iPhone, iPad) | ![](windows-media-server-or-web-server/_static/image29.gif) | ![](windows-media-server-or-web-server/_static/image30.gif) |
| Delivers content referenced by server-controlled playlists | ![](windows-media-server-or-web-server/_static/image31.gif) | **7![](windows-media-server-or-web-server/_static/image32.gif)** |
| Provides advanced performance boundary limits properties | ![](windows-media-server-or-web-server/_static/image33.gif) | **1![](windows-media-server-or-web-server/_static/image34.gif)** |

> | Supported on [Server Core](https://technet2.microsoft.com/windowsserver2008/en/library/47a23a74-e13c-46de-8d30-ad0afb1eaffc1033.mspx?mfr=true) installations | ![](windows-media-server-or-web-server/_static/image35.gif) | ![](windows-media-server-or-web-server/_static/image36.gif) |

Reviewer: ```suggestion
| Supported on Server Core installations | ![](windows-media-server-or-web-server/_static/image35.gif) | ![](windows-media-server-or-web-server/_static/image36.gif) |
```
